mate-panel/panel-widget.c

```diff
@@ -1693,7 +1690,7 @@ panel_widget_size_allocate(GtkWidget *widget, GtkAllocation *allocation)
 				prior_space = ad->constrained;
 				if (list->prev)
 				{
-					pad = list->prev->data;
+					AppletData *pad = list->prev->data;
```

Comment:
This gives me a build warning in travis-ci (cppcheck) https://app.travis-ci.com/github/mate-desktop/mate-panel/jobs/613338178#L5099
```
mate-panel/panel-widget.c:1693:18: style: Variable 'pad' can be declared as pointer to const [constVariablePointer]

     AppletData *pad = list->prev->data;

                 ^
```

Reply:
Should be fixed (although it's kind of unrelated, it's good to have less mutables)